### DISCUSSION TITLE
Using the `"FormKitConfig"` with `autoImport`

### DISCUSSION THREAD

**Discussion by gbyesiltas** [formkit/formkit#1072] [General]
Hey guys, I'm trying to use `autoImport` in nuxt and I have an issue. 

I have a language switcher which also changes the locale of FormKit ([this way](https://formkit.com/essentials/internationalization#using-root-config)) and I believe because at that point FormKit isn't injected/created I get a `injection "Symbol(FormKitConfig)" not found.` error. Does anyone have an idea about how to handle this?

[Original discord message](https://discord.com/channels/793529058377072650/940340536747896842/1183813133291241623)

**Comment by MiguelFranken** [comment]
@gbyesiltas Have you solved the issue?

**Reply by gbyesiltas** [reply]
Nope :(

**Comment by gbyesiltas** [comment]
Solved here :): https://github.com/formkit/formkit/commit/8771b6a0a3189d0a183abf91f2b93c713081ab91
